crates/pop-parachains/src/build.rs

```diff
@@ -1,30 +1,68 @@
 // SPDX-License-Identifier: GPL-3.0
 
-use crate::errors::Error;
+use crate::Error;
+use anyhow::Result;
 use duct::cmd;
-use std::path::Path;
+use pop_common::{parse_package_name, replace_in_file};
+use serde_json::Value;
+use std::{
+	collections::HashMap,
+	fs,
+	path::{Path, PathBuf},
+};
 
-/// Build the parachain located in the specified `path`.
+/// Enum representing the build profile for a parachain.
+#[derive(Debug, PartialEq)]
+pub enum Profile {
+	/// Debug profile, optimized for debugging.
+	Debug,
+	/// Release profile, optimized without any debugging functionality.
+	Release,
```

Comment:
Can add another one for `Other(String)` which can be used for things like the `production` profile.

This should be defined in `pop-common` imo.

Reply:
Agreed on moving it into `pop-common.` I think we should only add new profiles once needed, what's the benefit on adding it now?